src/NetCDF.jl

```diff
@@ -411,9 +419,9 @@ function create(name::AbstractString,varlist::Array{NcVar};gatts::Dict{Any,Any}=
 
   #Create the file
   id = nc_create(name,mode)
-
   # Collect Dimensions and set NetCDF ID
-  vars=Dict{ASCIIString,NcVar}();
+  # vars=Dict{ASCIIString,NcVar}();
+  vars = Dict{AbstractString,NcVar}();
```

Comment:
Why is this necessary? In general one should not use Abstract types in containers. Are there non-ASCII variable names? Would UTF8String be sufficient?